cmakebuild.js

```diff
@@ -0,0 +1,36 @@
+var spawn = require('child_process').spawn
+var which = require('npm-which')(process.cwd())
+
+function runCmake (opts, target, cb) {
+  which('cmake-js', function (err, cmakeJsPath) {
+    if (err) return cb(err)
+
+    var args = ['rebuild']
+    args.push('--runtime-version=' + target)
+    args.push('--target_arch=' + opts.arch)
+    args.push('--runtime=' + opts.runtime)
+
+    if (opts.debug) args.push('--debug')
+
+    var foundRest = false
+    for (var arg of process.argv) {
```

Comment:
This is a bit tricker to test. If you instead did:

```js
for (var arg of opts.args) {
  // ..
}
```

It would be easier to test this function.